docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/templates/content/article/node--cgov-article--full.html.twig

```diff
@@ -17,5 +17,9 @@
     <footer class="article-footer">
       {{ content.field_related_resources }}
       {{ contentPageMacros.displayDates(node, content) }}
+      {% set catch_cache = content.field_public_use|render %}
+      {% if content.field_public_use|field_raw('value') == '1' %}
```

Comment:
Set the FieldFormatter to be key for field_public_use and remove the catch_cache and set the check to just use field_value.